executor/insert_common.go

```diff
@@ -86,7 +86,7 @@ type InsertValues struct {
 
 	stats *InsertRuntimeStat
 
-	// LoadData use two goroutines. One for generate batch data,
+	// isLoadData LoadData use two goroutines. One for generate batch data,
```

Comment:
isLoadData indicates whatever current goroutine is use for generating batch data. LoadData use two goroutines...